[2025-07-07T19:55:25.038+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T19:55:13.776625+00:00 [queued]>
[2025-07-07T19:55:25.044+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T19:55:13.776625+00:00 [queued]>
[2025-07-07T19:55:25.044+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-07-07T19:55:25.056+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-07 19:55:13.776625+00:00
[2025-07-07T19:55:25.064+0000] {standard_task_runner.py:57} INFO - Started process 1778 to run task
[2025-07-07T19:55:25.067+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_activity_analysis', 'transform_data', 'manual__2025-07-07T19:55:13.776625+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/customer_activity_dag.py', '--cfg-path', '/tmp/tmpn3yosu3m']
[2025-07-07T19:55:25.069+0000] {standard_task_runner.py:85} INFO - Job 66: Subtask transform_data
[2025-07-07T19:55:25.082+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-07T19:55:25.107+0000] {task_command.py:410} INFO - Running <TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T19:55:13.776625+00:00 [running]> on host 7a17032e36f4
[2025-07-07T19:55:25.334+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ds_const@mail.ru' AIRFLOW_CTX_DAG_OWNER='Daniil' AIRFLOW_CTX_DAG_ID='customer_activity_analysis' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T19:55:13.776625+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T19:55:13.776625+00:00'
[2025-07-07T19:55:25.344+0000] {logging_mixin.py:150} INFO - /opt/***/shared_tmp/extract_manual__2025-07-07T19:55:13.776625+00:00.csv
[2025-07-07T19:55:26.483+0000] {logging_mixin.py:150} INFO - Файл /opt/***/shared_tmp/extract_manual__2025-07-07T19:55:13.776625+00:00.csv успешно удалён
[2025-07-07T19:55:26.483+0000] {logging_mixin.py:150} INFO - Received date type: <class 'datetime.datetime'>, value: 2024-02-01 00:00:00
[2025-07-07T19:55:27.023+0000] {logging_mixin.py:150} WARNING - Generating flags:   0%|          | 0/10 [00:00<?, ?it/s]
[2025-07-07T19:55:27.041+0000] {logging_mixin.py:150} WARNING - Generating flags: 100%|##########| 10/10 [00:00<00:00, 552.61it/s]
[2025-07-07T19:55:27.423+0000] {python.py:183} INFO - Done. Returned value was: /opt/***/shared_tmp/transform_manual__2025-07-07T19:55:13.776625+00:00.csv
[2025-07-07T19:55:27.446+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=customer_activity_analysis, task_id=transform_data, execution_date=20250707T195513, start_date=20250707T195525, end_date=20250707T195527
[2025-07-07T19:55:27.484+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-07-07T19:55:27.502+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
